tests/PHPUnit/PredisTestCase.php

```diff
@@ -333,8 +341,6 @@ protected function checkRequiredRedisServerVersion()
      */
     protected function checkRequirements()
```

Comment:
Can we drop this method and call `checkRequiredRedisServerVersion` directly in `setUp`?

Reply:
Makes sense.